README.md

```diff
@@ -1,132 +1,164 @@
 # SwiftFusion
 
-Differentiable Swift based sensor fusion library. 
+A differentiable sensor fusion library written in Swift.
 
-Think factor graphs a la [GTSAM](https://gtsam.org/) coupled with deep learning, [Jax](https://github.com/google/jax)/[TensorFlow](https://www.tensorflow.org/) style. Based on [Swift for TensorFlow](https://www.tensorflow.org/swift).
+Think factor graphs (à la [GTSAM](https://gtsam.org)) coupled with deep learning (via [Swift for TensorFlow](https://www.tensorflow.org/swift)).
 
-Still very early, but feel free to explore! Subject to *massive* change :-)
+This project is in an early phase, but feel free to explore! Subject to *massive* changes. :-)
 
-## Getting Started
+## Getting started
 
-### Swift on MacOS
+### Installation
 
-Using XCode is the easiest way to develop on Mac. Unfortunately, the current version of Xcode 12 has a bug, and you need to work with the XCode beta, available from Apple [here](https://developer.apple.com/download/).
+#### macOS
 
-Follow the instructions to [install Swift for TensorFlow on MacOS](https://github.com/tensorflow/swift/blob/master/Installation.md#macos). Installing the latest development snapshot is recommended.
+Using Xcode is the easiest way to develop on macOS. Unfortunately, the current version of Xcode 12 has a bug, and you need to work with the Xcode beta, available from Apple [here](https://developer.apple.com/download/).
+
+Download a [Swift for TensorFlow macOS toolchain](https://github.com/tensorflow/swift/blob/main/Installation.md#development-snapshots) and follow [these installation instructions](https://github.com/tensorflow/swift/blob/master/Installation.md#macos). Installing the latest development snapshot is recommended.
+
+To use Swift for TensorFlow toolchains:
 
-To use experimental toolchain:
 ```
-export PATH="/Library/Developer/Toolchains/swift-tensorflow-RELEASE-0.11.xctoolchain/usr/bin/:$PATH"
+export PATH="/Library/Developer/Toolchains/swift-tensorflow-RELEASE-0.12.xctoolchain/usr/bin/:$PATH"
 ```
 
-To re-generate XCode project:
+You can generate an Xcode project to open SwiftFusion in Xcode.
+
+Regenerating the Xcode project may be necessary if you add or remove Swift source files without using Xcode:
```

Comment:
Maybe we don't need this anymore. Let's delete all generation stuff cause this will cause confusion.

Reply:
Thanks for the note! [`swift package generate-xcodeproj` is indeed becoming deprecated](https://forums.swift.org/t/rfc-deprecating-generate-xcodeproj/42159), though folks sometimes fall back on it instead of using Xcode's SwiftPM support.

I moved the "Xcode development" section under the `## Development` section alongside `### Visual Studio Code`.